template/scripts/generateDocs.js

```diff
@@ -68,20 +79,15 @@ function formatTemplateFileContents(content, destination) {
 };
 
 function formatApiTableContents(content, destination) {
-  const nameExtractionData = nameExtraction();
-  const wcName = nameExtractionData.namespace + '-' + nameExtractionData.name;
-
   let result = content;
 
-  result = result
-    .replace(/\r\n|\r|\n####\s`([a-zA-Z]*)`/g, `\r\n#### <a name="$1"></a>\`$1\`<a href="#${wcName}" style="float: right; font-size: 1rem; font-weight: 100;">back to top</a>`)
-    .replace(/\r\n|\r|\n\|\s`([a-zA-Z]*)`/g, '\r\n| [$1](#$1)')
-    .replace(/\| \[\]\(#\)/g, "");
-
   fs.writeFileSync(destination, result, { encoding: 'utf8'});
 
-  fs.readFile('./demo/apiExamples.md', 'utf8', function(err, data) {
-    formatTemplateFileContents(data, './demo/apiExamples.md');
+  fs.readFile('./demo/api.md', 'utf8', function(err, data) {
+    formatTemplateFileContents(data, './demo/api.md');
```

Comment:
I get the following error traced back to this update

```
./index.js → dist/...
created dist/ in 266ms
/Users/dalesande/src/auro/test-element/scripts/generateDocs.js:49
  result = result.replace(/\[npm]/g, nameExtractionData.npm);
                  ^

TypeError: Cannot read properties of undefined (reading 'replace')
    at formatTemplateFileContents (/Users/dalesande/src/auro/test-element/scripts/generateDocs.js:49:19)
    at ReadFileContext.callback (/Users/dalesande/src/auro/test-element/scripts/generateDocs.js:87:5)
    at FSReqCallback.readFileAfterOpen [as oncomplete] (node:fs:325:13)

Node.js v18.15.0
ERROR: "build:docs" exited with 1.
```

Reply:
In what repo/branch did you see this?

Reply:
I generated a new repo using `npm run build:test`